coordinator/src/node.rs

```diff
@@ -87,23 +87,13 @@ impl Node {
         }
     }
 
-    /// Returns true or false, whether we can find an usable channel with the provided trader.
-    ///
-    /// Note, we use the usable channel to implicitely check if the user is connected, as it
-    /// wouldn't be usable otherwise.
-    pub fn is_connected(&self, trader: &PublicKey) -> bool {
-        let usable_channels = self.inner.channel_manager.list_usable_channels();
-        let usable_channels = usable_channels
+    /// Returns true or false, whether the given peer_id is connected with us.
+    pub fn is_connected(&self, peer_id: PublicKey) -> bool {
```

Comment:
:question: I guess we weren't using this for anything critical? Nice fix.

Reply:
It wasn't used at all.